src/backends/backend.js

```diff
@@ -277,6 +278,13 @@ class Backend {
       entry.data[filterRule.get('field')] === filterRule.get('value')
     ));
   }
+
+  supportsWorkflow() {
+    if(typeof this.implementation.supportsWorkflow === "function")
```

Comment:
I wonder if it would be better to just add this function to all the backends instead of this?

Reply:
Yeah not sure just thought it was more self contained this way at the time but since there are a few backends that don't support workflow now (unfortunately) then it makes sense to add it to all of them.  btw, I hope that we can implement workflow for bitbucket and gitlab sooner than later, I think we should create separate issues so they don't fall through the cracks.

Reply:
Yeah, I think we need to figure out which way is best so that we can keep our PR's the same.

Reply:
I prefer this approach, but it takes us halfway to what we really need, which is some sort of capabilities record for each backend, of which editorial workflow would be one. For now, I think @tech4him1's approach is ideal in its simplicity.

Reply:
Ah cool yeah that's much simpler code-wise to add a config value, although I'm not sure if it makes sense to put that on the user, it's one more cognitive thing they have to look up or know instead of it just being built into the backend automatically.  Any config is already daunting to most users and especially having to know internal details of supported features or not.  Also if we ever add workflow support it will just start working without having to tell everybody to change their config, update docs and so on..  you're guys call let me know and I'll update and push again if needed

Reply:
@zanedev The code that I had just checks to see if editorial workflow is enabled, the user doesn't have to have any other config value.

Reply:
sorry im mistaken it's not an actual config value ha, ok I'll update my code and push :)

Reply:
Well, it is an actual `config.yml` value, but it is just the one that is used to enable editorial workflow anyway (`publish_mode: editorial_workflow`). Do you think there is a better way?

Reply:
yep I was mistaken in my original comment.. already pushed a version that matches yours.

Reply:
OK, great. I think we should move to something more like what you had down the road, though, it just seems a little complicated for integrating right now.

Reply:
> btw, I hope that we can implement workflow for bitbucket and gitlab sooner than later, I think we should create separate issues so they don't fall through the cracks.

@zanedev I made an issue at #568 for discussing how to do EW on these backends.

Reply:
thanks i've been meaning to do that..